.gitignore

```diff
@@ -5,4 +5,5 @@ src/.vscode/launch.json
 *.code-workspace
 *.bin
 /log/
-.vscode/launch.json
\ No newline at end of file
+.vscode/launch.json
```

Comment:
These probably should live in .git/info/exclude instead

Reply:
I think its fine to leave IDE project folders here, should probably just ignore all of .vscode instead of the specific file though

https://github.com/rust-lang/rust/blob/master/.gitignore#L18